includes/models/model.llms.student.php

```diff
@@ -1145,20 +1146,21 @@ public function get_progress( $object_id, $type = 'course', $use_cache = true )
 	}
 
 	/**
-	 * Retrieve the Students original registration date in chosen format
+	 * Retrieve the student's original registration date in the chosen format.
 	 *
-	 * @param    string $format  any date format that can be passed to date()
-	 * @return   string
-	 * @since    ??
-	 * @version  3.14.0
+	 * @since ??
```

Comment:
We're slowly replacing all instances of this with "Unknown"

```suggestion
	 * @since Unknown
```